### PR TITLE
Added MoCoGAN

### PR DESCRIPTION
**Pull request by ayush12gupta**
deleted the cGAN from pull req

### CODE REVIEW

File: generative_models/MoCoGAN_PyTorch/main.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    
+    # Making required folder
+    if not os.path.exists('./generated_videos'):
+      os.makedirs('./generated_videos')
+    if not os.path.exists('./trained_models'):
+      os.makedirs('./trained_models')
+    if not os.path.exists('./resized_data'):
+      os.makedirs('./resized_data')
+
+    T = 16
```
Comment (nirmal-suthar) [inline generative_models/MoCoGAN_PyTorch/main.py]: add this to parser

File: generative_models/MoCoGAN_PyTorch/main.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+    start_epoch = 1
+    seed = 0
+    torch.manual_seed(seed)
+    np.random.seed(seed)
```
Comment (nirmal-suthar) [inline generative_models/MoCoGAN_PyTorch/main.py]: no need to give manual seed

File: generative_models/MoCoGAN_PyTorch/main.py [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    parser.add_argument('--pre-train', type=int, default=-1,
+                     help='set 1 when you use pre-trained models'),
+    parser.add_argument('--img_size', type=int, default=96),
+    parser.add_argument('--channel', type=int, default=3),
+    parser.add_argument('--hidden', type=int, default=100),
+    parser.add_argument('--dc', type=int, default=50),
+    parser.add_argument('--de', type=int, default=10),
+    parser.add_argument('--lr', type=int, default=0.0002),
+    parser.add_argument('--beta', type=int, default=0.5),
+    parser.add_argument('--trained_path', type=str, default='trained_models')
```
Comment (nirmal-suthar) [inline generative_models/MoCoGAN_PyTorch/main.py]: describe the usage of each args (help argument)

**Review by nirmal-suthar — APPROVED** [review]

**Comment by nirmal-suthar** [comment]
also update usage in README as well

**Comment by ayush12gupta** [comment]
Did all the changes

**Pull request merged**